docs/source/Plugin/P053.rst

```diff
@@ -139,17 +139,17 @@ Available options:
 
 .. image:: P053_OutputValuesOptions.png
 
-*Particles µg/m3: pm1.0, pm2.5, pm10* The default output values, available for all supported sensors.
+*Particles /m3: pm1.0, pm2.5, pm10* The default output values, available for all supported sensors.
 
-- PM1.0 - Concentration of particles > 1 micron (µm/m3)
-- PM2.5 - Concentration of particles > 2.5 micron (µm/m3)
-- PM10 - Concentration of particles > 10 micron (µm/m3)
+- PM1.0 - Concentration of particles > 1 µm/m3
+- PM2.5 - Concentration of particles > 2.5 µm/m3
+- PM10 - Concentration of particles > 10 µm/m3

```

Comment:
The `micron` was OK, just that `µm/m3` really looks odd.
The SDS011 does report it in `µg/m3`

Reply:
Ah, yes, this one also does report in µg/m3 for the PM measurements, it's the Count measurements that report the particles of specific sizes per 0.1L that confused my sight. Will correct.